To: vim_dev@googlegroups.com
Subject: Patch 7.3.189
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.189 (after 7.3.186)
Problem:    Can't build without +clipboard feature. (Christian Ebert)
Solution:   Add the missing #ifdef.
Files:	    src/normal.c


*** ../vim-7.3.188/src/normal.c	2011-05-10 16:12:40.000000000 +0200
--- src/normal.c	2011-05-10 17:25:26.000000000 +0200
***************
*** 1204,1212 ****
--- 1204,1215 ----
  #ifdef FEAT_EVAL
  	{
  	    int regname = 0;
+ 
  	    /* Adjust the register according to 'clipboard', so that when
  	     * "unnamed" is present it becomes '*' or '+' instead of '"'. */
+ # ifdef FEAT_CLIPBOARD
  	    adjust_clip_reg(&regname);
+ # endif
  	    set_reg_var(regname);
  	}
  #endif
*** ../vim-7.3.188/src/version.c	2011-05-10 17:21:34.000000000 +0200
--- src/version.c	2011-05-10 17:29:21.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     189,
  /**/

-- 
If your nose runs, and your feet smell, you might be upside down.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
